src/inc/gcdecoder.cpp

```diff
@@ -92,12 +92,11 @@ PTR_CBYTE FASTCALL decodeHeader(PTR_CBYTE table, UINT32 version, InfoHdr* header
 
     BYTE nextByte = *table++;
     BYTE encoding = nextByte & 0x7f;
-    const BYTE maskHaveMoreBytesBit = MORE_BYTES_TO_FOLLOW - 1;
```

Comment:
This change is independent of allowing V1 for desktop.
It implements a change @briansull asked in this review: https://github.com/dotnet/coreclr/pull/6974#r78816707